modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -461,7 +486,7 @@ protected Page<ObjectEntry> getObjectEntries(
 	}
 
 	private ObjectEntry _addObjectEntry(
-			String customStatus, Date date, String title)
+			String customStatus, Date date, String title, boolean flagged)
```

Comment:
Hi @jairmedeiros, `flagged` parameter should come before `title`, please sort this.


Thanks!